src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -404,6 +418,42 @@ def _get_linux_fx_version(cmd, resource_group_name, name, slot=None):
     return site_config.linux_fx_version
 
 
+def url_validator(url):
+    try:
+        result = urlparse(url)
+        return all([result.scheme, result.netloc, result.path])
+    except:
+        return False
+
+
+def _get_linux_multicontainer_config_file(cmd, resource_group_name, name, file_name=None, encoded=True, slot=None):
+    from base64 import b64encode, b64decode
+    config_file_bytes = None
+    if file_name:
+        if url_validator(file_name):
+            from urllib.request import urlopen
+            response = urlopen(file_name)
+            config_file_bytes = response.read()
+        else:
+            with open(file_name) as f:
```

Comment:
since you will encode back in 2 lines below, you are better off just reading the file in binary mode, so why not `with open(file_name, 'rb')`?